src/Toolchain/ToolchainEnv.ts

```diff
@@ -170,14 +183,20 @@ class ToolchainEnv extends Env {
     return new Promise<boolean>((resolve, reject) => {
       const jobs: Array<Job> = [];
       const job = new JobUninstall(toolchain.uninstall());
-      job.successCallback = () => resolve(true);
+      job.successCallback = () => {
+        // TODO(jyoung) What toolchain will be default item?
```

Comment:
Is this comment related with uninstalling...? @.@

Reply:
AFAIU,

>>> After uninstalling, which toolchain should be as default?

Reply:
If the default toolchain is uninstalled, the default toolchain should be changed to another one.

Reply:
Ah... I thought that `undefined` is proper to be set after uninstalling.
If other default toolchain should be set after uninstalling, this comment is valid :)